### ISSUE TITLE
HTTP Error 500.19 on local iis server

### ISSUE THREAD

**Issue by titans2016** [NuGet/NuGetGallery#3075]
Hi.
I downloaded source code and want upload to local iis.
I have installed SQL but first I want that site show me main page and then configure connection string.
In NugetGallery I set up Local IIS: http://localhost/NuGetGallery.
In application pool I setup Identity under administrator user.
This project successfully build on my machine.
But when I open site I see:
 
HTTP Error 500.19 - Internal Server Error

The requested page cannot be accessed because the related configuration data for the page is invalid.

Detailed Error Information:

Module
   IIS Web Core 

Notification
   BeginRequest 

Handler
   Not yet determined 

Error Code
   0x8007000d 

Config Error
    

Config File
   \\?\D:\Nuget\src\src\NuGetGallery\web.config 

What should I change to fix this issue?

**Comment by maartenba** [comment]
Can you try running on a development machine first, following steps in readme? Also check iis logs/event viewer if possible. And make sure to turn off windows authentication ☺

**Comment by titans2016** [comment]
I try to build with build.cmd. But it show me error:

>   C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1820,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [D:\Nuget\src\src\NuGetGallery.Operations\NuGetGallery.Operations.csproj]


"D:\Nuget\src\build.msbuild" (default target) (1) ->
"D:\Nuget\src\packages\NuGet.Services.Build.3.0.13\tools\NuGet.Services.FullBuild.msbuild" (default target) (2) ->
"D:\Nuget\src\NuGetGallery.sln" (Build target) (3) ->
"D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj" (default target) (7) ->
(RunFacts target) -> 
  D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error : NuGetGallery.SearchClient.RetryingHttpClientWrapperFacts.FailsWhenNoValidUriGiven1: Assert.Throws() Failure\r\nExpected: typeof(System.AggregateException)\r\nActual:   (No exception was thrown) [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
  D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error : --- End of stack trace from previous location where exception was thrown ---\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error :    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error :    at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error :    at NuGetGallery.SearchClient.RetryingHttpClientWrapperFacts.<FailsWhenNoValidUriGiven1>d__14.MoveNext() in D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs:line 129\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error : --- End of stack trace from previous location where exception was thrown ---\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error :    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error : --- End of stack trace from previous location where exception was thrown ---\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error :    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error : --- End of stack trace from previous location where exception was thrown ---\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error :    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(129): error :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
  D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error : NuGetGallery.SearchClient.RetryingHttpClientWrapperFacts.FailsWhenNoValidUriGiven2: Assert.Throws() Failure\r\nExpected: typeof(System.AggregateException)\r\nActual:   (No exception was thrown) [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
  D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error : --- End of stack trace from previous location where exception was thrown ---\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error :    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error :    at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error :    at NuGetGallery.SearchClient.RetryingHttpClientWrapperFacts.<FailsWhenNoValidUriGiven2>d__15.MoveNext() in D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs:line 137\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error : --- End of stack trace from previous location where exception was thrown ---\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error :    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error : --- End of stack trace from previous location where exception was thrown ---\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error :    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error : --- End of stack trace from previous location where exception was thrown ---\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error :    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(137): error :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
  D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(72): error : NuGetGallery.SearchClient.RetryingHttpClientWrapperFacts.ReturnsSuccessResponseForCollectionContainingValidUri: Assert.True() Failure\r\nExpected: True\r\nActual:   False [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
  D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(72): error :    at NuGetGallery.SearchClient.RetryingHttpClientWrapperFacts.<ReturnsSuccessResponseForCollectionContainingValidUri>d__11.MoveNext() in D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs:line 72\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(72): error : --- End of stack trace from previous location where exception was thrown ---\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(72): error :    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(72): error :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(72): error : --- End of stack trace from previous location where exception was thrown ---\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(72): error :    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(72): error :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(72): error : --- End of stack trace from previous location where exception was thrown ---\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(72): error :    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]
D:\Nuget\src\tests\NuGetGallery.Facts\SearchClient\RetryingHttpClientWrapperFacts.cs(72): error :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) [D:\Nuget\src\tests\NuGetGallery.Facts\NuGetGallery.Facts.csproj]

    2 Warning(s)
    6 Error(s)

I can build this code in visual studio 2015 but without msbuild argument /tv:14.0 /p:VisualStudioVersion=14.0  /p:ToolsVersion=14.0 

I took this code and it's doesn't build. 
How it's fix?

**Comment by maartenba** [comment]
Make sure you have VS2015.2 installed, and latest .NET versions. Then run build.cmd from the `dev` branch.

**Comment by titans2016** [comment]
My last .net 4.6.2.  My version of visual studio 14.0.25123.00 Update 2.
Failed with same error.

**Comment by maartenba** [comment]
Can you clear the packages folder and all bin/obj folders? (or run git clean -xdf)

**Comment by titans2016** [comment]
Same trouble.
Maybe it require .Net 4.5.2?
I see in msbuild log (for example part):
          For SearchPath "{TargetFrameworkDirectory}". (TaskId:24)
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5.2\System.Core.winmd", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5.2\System.Core.dll", but its name "System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" didn't match. (TaskId:24)
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5.2\System.Core.exe", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5.2\Facades\System.Core.winmd", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5.2\Facades\System.Core.dll", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5.2\Facades\System.Core.exe", but it didn't exist. (TaskId:24)
          For SearchPath "{Registry:Software\Microsoft\.NETFramework,v4.5.2,AssemblyFoldersEx}". (TaskId:24)
          Considered AssemblyFoldersEx locations. (TaskId:24)
          For SearchPath "{AssemblyFolders}". (TaskId:24)
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.5\System.Core.winmd", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.5\System.Core.dll", but its name "System.Core, Version=3.5.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" didn't match. (TaskId:24)
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.5\System.Core.exe", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Microsoft SQL Server\130\SDK\Assemblies\System.Core.winmd", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Microsoft SQL Server\130\SDK\Assemblies\System.Core.dll", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Microsoft SQL Server\130\SDK\Assemblies\System.Core.exe", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Microsoft SQL Server\120\SDK\Assemblies\System.Core.winmd", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Microsoft SQL Server\120\SDK\Assemblies\System.Core.dll", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Microsoft SQL Server\120\SDK\Assemblies\System.Core.exe", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files\IIS\Microsoft Web Deploy V3\System.Core.winmd", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files\IIS\Microsoft Web Deploy V3\System.Core.dll", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files\IIS\Microsoft Web Deploy V3\System.Core.exe", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Microsoft.NET\ADOMD.NET\120\System.Core.winmd", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Microsoft.NET\ADOMD.NET\120\System.Core.dll", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Microsoft.NET\ADOMD.NET\120\System.Core.exe", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.0\System.Core.winmd", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.0\System.Core.dll", but it didn't exist. (TaskId:24)
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.0\System.Core.exe", but it didn't exist. (TaskId:24)
          For SearchPath "{GAC}". (TaskId:24)
          Considered "System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e, Retargetable=Yes", which was not found in the GAC. (TaskId:24)
          For SearchPath "bin\Debug\". (TaskId:24)
          Considered "bin\Debug\System.Core.winmd", but it didn't exist. (TaskId:24)
          Considered "bin\Debug\System.Core.dll", but it didn't exist. (TaskId:24)
          Considered "bin\Debug\System.Core.exe", but it didn't exist. (TaskId:24
         
Considered "D:\Nuget\dev\packages\entityframework.6.1.3\lib\net45\System.Runtime.Serialization.dll", but it didn't exist. 

          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5.2\Newtonsoft.Json.dll", but it didn't exist.

Considered "D:\Nuget\dev\packages\Elmah.Contrib.Mvc.1.0\lib\net40\Elmah.dll", but it didn't exist.

          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5.2\msshrtmi.dll", but it didn't exist.

So it missing huge list of all libraries.
I haven't idea how it's fix.

**Comment by maartenba** [comment]
True, as well as the Azure SDK installed (2.9)

**Comment by titans2016** [comment]
Yes. I have installed version Azure SDK 2.9.1. Should I install to older?

**Comment by maartenba** [comment]
Can you try? Als .NET 4.5.2 may be needed.
If all else fails, run a git clean -xdf again and try openign the solution in Visual Studio and build from there, this may give more detailed errors.

**Comment by titans2016** [comment]
As I mention earlier in visual studio it's build perfectly success!

**Comment by maartenba** [comment]
Then you are good to go :)

**Comment by titans2016** [comment]
But when create a package from visual studio I can't use it IIS, because it show me error:HTTP Error 500.19 - Internal Server Error. Msdn said : Delete the malformed XML element from the ApplicationHost.config file or from the Web.config file. 
So, when I build via Visual Studio in malformed web config?

**Comment by maartenba** [comment]
You may want to double check your IIS configuration etc. to get it running in there - here are some tips that use request tracing - https://blogs.msdn.microsoft.com/webtopics/2010/03/08/troubleshooting-http-500-19-errors-in-iis-7/

**Comment by robertmuehsig** [comment]
Is the ASP.NET feature enabled for your IIS? Can you run other ASP.NET applications? 
In IIS: Your NuGet website runs with the correct .NET 4.0 AppPool?

**Comment by titans2016** [comment]
Yes. I have Server 2012 R2. And Asp.Net 4.5 was installed.
Yes. I can run another asp.Net application on IIS.
I use default app pool v4.0 under user who is admin on server.

**Comment by robertmuehsig** [comment]
Mhh... WindowsAuth is disabled? (sometimes, when WindowsAuth is enabled in IIS but disabled via Web.config strange things occure)
Could you post a screenshot from your IIS-Manager - something like this https://cloud.githubusercontent.com/assets/756703/15910712/7c8c64d8-2dcc-11e6-85a6-ac152b15c5b9.png ?

**Comment by titans2016** [comment]
Ok. So here screen when I try open windows authenticate.
[https://s31.postimg.org/a6llrp4xn/2016_06_15_16_16_19_10_80_82_127_Remote_Deskto.png](url)
And here screen of virtual site 
[https://s32.postimg.org/nrnmkea51/2016_06_15_16_16_42_10_80_82_127_Remote_Deskto.png](url)

**Comment by titans2016** [comment]
As I can see WindowsAuth is disabled.

**Comment by robertmuehsig** [comment]
Mhh - I would guess that the Default Web Site does have a web.config as well, which might be conflicting with your NuGetGallery. Or maybe IIS can't access the path.

Could you try to create a new Website directly under "Sites" and point to your NuGet installation and see if that works? (you might need to disabled the Default Web Site or change the port).

**Comment by titans2016** [comment]
Default web site hasn't web config. It'e default site. I create new web site and see same issue:
[https://s32.postimg.org/73poycwpx/2016_06_15_16_38_25_10_80_82_127_Remote_Deskto.png](url)

**Comment by robertmuehsig** [comment]
And when you access the Authentication-Settings in IIS you see this "There was an error..." message? If yes, then I would guess that IIS can't access web.config & co.
If you could zip the folder I would try it on my machine at home.

**Comment by titans2016** [comment]
Yes. This is exactly this error. 
Sure. Can you download by this google driver[https://drive.google.com/file/d/0B1cf9uXeYIUVblpSTnN4UzBsNk0/view?usp=sharing](url)

Also I put instead web config, another web config from default site on Visual Studio MVC project, and it show me another error. but no problem with access.

**Comment by robertmuehsig** [comment]
Mhh... I couldn't reproduce the issue with your zip.

These are the steps I did:

* Download the zip
* Unblock the zip (otherwise dlls are not getting loaded)
* Add a new Web Application beneth the Default Web Site with DefaultAppPool, pointing to "C:\Users\Robert\Downloads\Nuget\src\src\NuGetGallery"

![image](https://cloud.githubusercontent.com/assets/756703/16099084/aa4a2b14-3356-11e6-9dc1-412c1bb437c4.png)

* For simplicity I just gave "Everyone" full access to the folder

![image](https://cloud.githubusercontent.com/assets/756703/16099110/cf3f7ca8-3356-11e6-90c3-248809bb00fe.png)

Result:
![image](https://cloud.githubusercontent.com/assets/756703/16099119/dcc45b50-3356-11e6-91e3-c289275cf3e6.png)

BTW: Just pointing to the NuGetGallery source is not really a good idea, because web.config transformations might not be applied that way. A cleaner way would be to "publish" it (can be done via VS).
